dali/python/nvidia/dali/_multiproc/pool.py

```diff
@@ -241,20 +250,22 @@ def scheduled_ahead(self):
 
 class WorkerContext:
 
-    def __init__(self, source_descs : SourceDescription, dedicated_task_queue : Optional[ShmQueue],
-                 shm_chunks : List[BufShmChunk]):
+    def __init__(self, source_descs: SourceDescription,
+                 dedicated_task_queue: Optional[ShmQueue], shm_chunks: List[BufShmChunk]):
```

Comment:
I guess at this point we can go with argument per line.
```suggestion
    def __init__(self,
                 source_descs: SourceDescription,
                 dedicated_task_queue: Optional[ShmQueue],
                 shm_chunks: List[BufShmChunk]):
```